publication-commons/src/main/java/no/unit/nva/publication/model/business/UserInstance.java

```diff
@@ -92,7 +92,10 @@ public static UserInstance fromDoiRequest(DoiRequest doiRequest) {
     }
 
     public static UserInstance fromPublication(Publication publication) {
-        return UserInstance.create(publication.getResourceOwner(), publication.getPublisher().getId());
+        return new UserInstance(publication.getResourceOwner().getOwner().getValue(),
+                                publication.getPublisher().getId(),
+                                publication.getResourceOwner().getOwnerAffiliation(),
+                                null, List.of(), UserClientType.INTERNAL);
     }
```

Comment:
This is the same as we do today in addition to setting this: `publication.getResourceOwner().getOwnerAffiliation()`